packages/flutter_tools/lib/src/commands/config.dart

```diff
@@ -39,6 +40,9 @@ class ConfigCommand extends FlutterCommand {
   final List<String> aliases = <String>['configure'];
 
   @override
+  bool get shouldUpdateCache => true;
```

Comment:
Urk - no. This was a copy-paste error - this should be disabling shouldUpdateCache for `flutter config`.